decidim-proposals/config/locales/es.yml

```diff
@@ -52,6 +52,7 @@ es:
             proposal_edit_before_minutes: Las propuestas pueden ser editadas por los autores antes de que pasen estos minutos
             proposal_limit: Límite de propuestas por usuario
             vote_limit: Límite de votos por usuario
+            proposal_length: Tamaño máximo del cuerpo de la propuesta
```

Comment:
Please, do not modify locale files other than English ones. Translations are handled automatically.